e2etests/run_athens.go

```diff
@@ -0,0 +1,56 @@
+// +build e2etests
+
+package e2etests
+
+import (
+	"fmt"
+	"net/http"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"time"
+)
+
+func buildAthens(goBin string, destPath string, env []string) (string, error) {
+	target := path.Join(destPath, "athens-proxy")
+	binFolder, err := filepath.Abs("../cmd/proxy")
+	if err != nil {
+		return "", fmt.Errorf("Failed to get athens source path %v", err)
+	}
+
+	cmd := exec.Command(goBin, "build", "-o", target, binFolder)
+	cmd.Env = env
```

Comment:
I'd add a bytes.Buffer to cmd.Stderr and return it in case of error because then it will explain why go build failed.